corehq/apps/app_manager/templates/app_manager/app_diff.html

```diff
@@ -19,6 +19,9 @@
   {% registerurl 'compare_multimedia_sizes' app.domain app.id other_app.id '---' %}
   <div id="multimedia-sizes-diff-container">
     <h2>{% trans 'Multimedia Sizes' %}</h2>
+    {% if build_profiles %}
```

Comment:
`build_profiles_select.html` would be better self-contained if it also contained the `if build_profiles`.

Reply:
https://github.com/dimagi/commcare-hq/pull/26032/commits/d0132c2b0e42a96e06873518789d6af4e670339e